include_core/omrport.h

```diff
@@ -2209,6 +2209,8 @@ typedef struct OMRPortLibrary {
 	uintptr_t (*str_ftime)(struct OMRPortLibrary *portLibrary, char *buf, uintptr_t bufLen, const char *format, int64_t timeMillis) ;
 	/** see @ref omrstr.c::omrstr_ftime_ex "omrstr_ftime_ex"*/
 	uintptr_t (*str_ftime_ex)(struct OMRPortLibrary *portLibrary, char *buf, uintptr_t bufLen, const char *format, int64_t timeMillis, uint32_t flags);
+	/** see @ref omrstr.c::omrstr_current_time_zone "omrstr_current_time_zone"*/
+	int32_t (*str_current_time_zone)(struct OMRPortLibrary *portLibrary, int32_t *secondsEast, char *zoneNameBuffer, uint32_t zoneNameBufferLen) ;
```

Comment:
The type of `zoneNameBufferLen` should be `size_t` (see the proposed uses in eclipse-openj9/openj9#14293).